test/Specs/TextManipulationSpec.hs

```diff
@@ -0,0 +1,38 @@
+{-# LANGUAGE QuasiQuotes #-}
```

Comment:
While I remember this, can you grab the file `test/Property/NonEmptyStringsSpec.hs` from the backend code base and port the non-Persistent parts over here? I just remembered we haven't done this yet.

Reply:
Nice, I did not see that